src/tentaclio/clients/databricks_client.py

```diff
@@ -20,11 +25,22 @@
 
 
 class DatabricksClient(sqla_client.SQLAlchemyClient):
-    """Databricks client, backed by a pyodbc + SQLAlchemy connection"""
+    """Databricks client, backed by a pyodbc + SQLAlchemy connection."""
 
     def __init__(self, url: Union[str, urls.URL]) -> None:
+        """Create sqlalchemy client based on the passed url.
+
+        This is a wrapper for sqlalchemy engine/connection creation.
+        """
         super().__init__(url)
         self.driver = _get_simba_driver_path()
+        self.http_path = self._get_http_path()
+
+    def _get_http_path(self) -> str:
+        http_path = os.environ.get("TENTACLIO__DATABRICKS_HTTP_PATH", "")
+        if http_path == "" and self.url.query:
+            http_path = self.url.query["http_path"]
```

Comment:
Unsure of whether to raise an exception if http_path is missing or just let sqlalc handle it

Reply:
We do raise in other clients when there's parts of the URL that are required that are missing (eg HTTPClient raises `raise exceptions.HTTPError("Missing URL hostname")` when there's a missing hostname)